### PR TITLE
Fix radiance units in SEVIRI L1.5 nat/nc and FCI L1c nc readers

### PR DESCRIPTION
**Pull request by ameraner**
This PR fixes wrong radiance unit labels in the yaml files of the readers
- `seviri_l1b_native`
- `seviri_l1b_nc`
- `fci_l1c_nc`

All the readers had units of radiance per unit wavelength, but the files contain radiances per unit wavenumber.

The PR also makes sure that the FCI reader is not overwriting the yaml unit string with the content of the file. It was discussed and decided at the PCW that the unit inside Satpy shall be indicated as "mW m-2 sr-1 (cm-1)-1", rather than "mW.m-2.sr-1.(cm-1)-1" as indicated inside the file, for consistency with other Satpy readers and to better comply with other conventions.

In addition, the `radiance_unit_conversion_coefficient` is added to the attributes of FCI datasets loaded as radiances, to allow the user to convert between radiances.

PS: The tests of all readers indicated above check that the reader outputs the unit labels contained in the yaml, so no new test has been added (only the FCI one has been updated).

*Additional information*: 
Screenshot of the SEVIRI format description confirming that the data is in units of radiance per unit wavenumber:
image

Screenshot of the FCI L1 Product User Guide confirming that the data is in units of radiance per unit wavenumber:
image

 - [x] Fully documented

### CODE REVIEW

File: satpy/readers/fci_l1c_nc.py [hunk 1]
```diff
@@ -441,7 +446,7 @@ def calibrate(self, data, key):
         if key['calibration'] == "counts":
             # from package description, this just means not applying add_offset
             # and scale_factor
-            data.attrs["units"] = "1"
+            pass
         elif key['calibration'] in ['brightness_temperature', 'reflectance', 'radiance']:
             data = self.calibrate_counts_to_physical_quantity(data, key)
         else:
```
Comment (mraspaud) [inline satpy/readers/fci_l1c_nc.py:452]: How about
```python
        if key['calibration'] in ['brightness_temperature', 'reflectance', 'radiance']:
            data = self.calibrate_counts_to_physical_quantity(data, key)
        elif key['calibration'] != "counts":
...
```

**Review by mraspaud — COMMENTED** [review]
LGTM, just one minor suggestion

File: satpy/readers/fci_l1c_nc.py (hunk 1 above)
Comment (ameraner) [reply satpy/readers/fci_l1c_nc.py:452]: Yeah, that would work too. I kept the `pass` just to make clear that we don't do any calibration operation, but of course it's an unnecessary `if` branch in the end. I'll simplify the code as suggested.

File: satpy/readers/fci_l1c_nc.py [hunk 2]
```diff
@@ -267,6 +269,9 @@ def _get_dataset_measurand(self, key, info=None):
             res.attrs.pop("scale_factor")
             res.attrs.pop("warm_scale_factor")
 
+        if key['calibration'] == 'radiance':
```
Comment (sfinkens) [inline satpy/readers/fci_l1c_nc.py:272]: Would it make sense to do this in `calibrate_counts_to_rad`?

**Review by sfinkens — COMMENTED** [review]
Thanks for fixing this! Obviously I forgot the Native and netCDF readers in #490. Just one question.

File: satpy/readers/fci_l1c_nc.py (hunk 2 above)
Comment (ameraner) [reply satpy/readers/fci_l1c_nc.py]: good point, done :)

**Review by sfinkens — APPROVED** [review]
LGTM!

**Review by mraspaud — APPROVED** [review]
LGTM

**Pull request merged**